src/templates/_partials/_blog_post.twig

```diff
@@ -0,0 +1,16 @@
+<div class="pt-8 border-b border-gray-300 first:pt-0">
+  <p class="text-base leading-[19px] py-2 text-gray-700 font-semibold">
+    {{ blog.date }}
+  </p>
```

Comment:
Make sure use the `time` tag:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time